resources/pt/cemu.po

```diff
@@ -849,20 +831,21 @@ msgid ""
 "\n"
 "Recommended: Auto"
```

Comment:
'Auto' should be 'Automático' since you've changed it.

Reply:
I think you mentioned the wrong line here, but this made me find another translation I forgot to fix. Thanks!